src/core/lib/transport/metadata_batch.h

```diff
@@ -114,6 +136,20 @@ grpc_error* grpc_metadata_batch_add_tail(
     grpc_metadata_batch* batch, grpc_linked_mdelem* storage,
     grpc_mdelem elem_to_add) GRPC_MUST_USE_RESULT;
 
+inline grpc_error* GRPC_MUST_USE_RESULT grpc_metadata_batch_add_tail(
```

Comment:
Similar question to above.

Reply:
No idea why they both exist, tbh. They're basically identical. I'll ask @roth if there's any particular reason they may both still be there.